drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/java/org/drools/workbench/screens/scenariosimulation/backend/server/util/ScenarioBeanUtil.java

```diff
@@ -68,6 +68,10 @@ private ScenarioBeanUtil() {
     }
 
     public static Object navigateToObject(Object rootObject, List<String> steps) {
```

Comment:
Idea says it is not used anywhere, probably we are fine to remove?

Reply:
I did this to simplify potential code merge with current branches. If in one of the current development branches someone used the old call, this method expose previous signature with previous behaviour